### DISCUSSION TITLE
String Datatype

### DISCUSSION THREAD

**Discussion by mishelshaji** [mishelshaji/Learn-DotNet#9] [Progress Tracker]
Topics covered:
👉 What is a string?
👉 Creating a string.
👉 Accessing characters from a string using the index.
👉 Finding the length of a string.
👉 Looping over a string using for and foreach loop.
👉 String functions: Upper, Lower, Index, IndexOf, Contains, Trim, TrimStart, TrimEnd etc...
👉 String formatting and building: String concatenation, Interpolation, string.Format, string.Concat, string.Join, StringBuilder etc...

**Comment by ManikuttyBabu** [comment]
completed

**Comment by PEFathimaManfa** [comment]
completed

**Comment by Arjunprasad-22** [comment]
Completed

**Comment by Arun-babumon** [comment]
Completed

**Comment by SonuSanilK** [comment]
Completed

**Comment by sreelakshmiChandran** [comment]
Completed

**Comment by Nandana-04** [comment]
Completed

**Comment by Jiby-Joseph-123** [comment]
Completed.

**Comment by riya45810** [comment]
Completed.

**Comment by akhilamp123** [comment]
Completed

**Comment by sarath534** [comment]
Completed

**Comment by jeweljose22** [comment]
Completed

**Comment by kavitha-BS** [comment]
Completed

**Comment by AnitaMaryThomas** [comment]
Completed

**Comment by SivapriyaVB** [comment]
Completed
